controllers/topology/util_test.go

```diff
@@ -21,9 +21,8 @@ import (
 
 	"github.com/pkg/errors"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha4"
-	"sigs.k8s.io/cluster-api/internal/testtypes"
+	. "sigs.k8s.io/cluster-api/internal/testtypes"
```

Comment:
I'm not a fan of dot imports to be honest :/ (because of the "magic-factor")
I understand why we want it for matchers, I'm not sure if we should also use it for our builder funcs.

Reply:
Yeah - it's a compromise here from having one package with the two different functions. We are using . imports across our testing for Gomega. Do you have strong feelings about this? I can definitely see both sides.

Reply:
I'll move it to the new PR